src/main/java/minicraft/entity/mob/Sheep.java

```diff
@@ -46,7 +45,14 @@ public void render(Screen screen) {
 	@Override
 	public void tick() {
 		super.tick();
-		if (age - ageWhenCut > WOOL_GROW_TIME) cut = false;
+		if (random.nextInt(1000) == 0) { // Grazing
```

Comment:
I think that it is better to have the random check inside the `tile instanceof GrassTile` if-statement. This makes it so that the sheep has to find a grass tile first, and then check if it wants to eat it.

Reply:
I think it would be just fine. I would design it as when it wants to graze, first checking if there is any valid interactable tile under it, grazing it if so.

Reply:
First, you have to execute many more calls to random that you have to. Generating pseudorandom is more expensive than checking for a type. Secondly this makes it so that it practically never grazes, because the sheep has to be on a tile that it is able to graze on, which isn't certain.